### DISCUSSION TITLE
Is it possible to have an /index route?

### DISCUSSION THREAD

**Discussion by stefbowerman** [nuxt/nuxt#9836] [Questions (Nuxt 2)] [answered]
I'm being asked to create a website with an `/index` route that displays a list of references on the site, much like the index in the back of a book.

Since "index" is a special term within the routing system, I was wondering if anyone knows how to get the `/index` route to work as a normal subpage.

I tried creating a directory structure like this:

```
- /pages
  - index.vue
  - index/
    - index.vue
```
but that just creates a nested route since the pages directory has both a folder and file with the same name. If I check the generated routes in `.nuxt/routes.json` I see this:
```json
  {
    "path": "/",
    "component": "/site/pages/index.vue",
    "chunkName": "pages/index",
    "children": [
      {
        "name": "index",
        "path": "",
        "component": "/site/pages/index/index.vue",
        "chunkName": "pages/index/index",
        "_name": "_a31953b2"
      }
    ],
    "_name": "_15545aa4"
  }
```
I realize that the easy solution would be to come up with a different name for the route but I was just wondering if there's any way around this router limitation?

Thanks

**Comment by kissu** [comment, marked as answer]
You could use this module: https://github.com/nuxt-community/router-extras-module
And set the `path` or even an `alias` key (if you want to have both) and keep a single `index.vue`, which is usually the index (`/`).

There are other cool options with it. 👌🏻

**Reply by stefbowerman** [reply]
Thank you this is exactly what I needed!
